nextflow.config

```
@@ -1,22 +1,166 @@
 /*
-========================================================================================
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     nf-core/quantms Nextflow config file
-========================================================================================
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     Default config options for all compute environments
 ----------------------------------------------------------------------------------------
 */
 
 // Global default params, used in configs
 params {
 
-    // TODO nf-core: Specify your pipeline's command line flags
-    // Input options
-    input                      = null
+    // Workflow flags
+    input = null // the sdrf and spectra parameters are inferred from this one
+    root_folder = null
+    local_input_type = null
+    database = null
+    acquisition_method = null
```

You could make that nicer (as usually recommended), i.e.
```suggestion
    input                        = null // the sdrf and spectra parameters are inferred from this one
    root_folder               = null
    local_input_type      = null
    database                 = null
    acquisition_method = null
```
also true for all entries below.